docs/source/tutorial/detection.rst

```diff
@@ -0,0 +1,176 @@
+Object Detection Tutorial
+=========================
+
+This tutorial will walk you through the features related to object detection that ChainerCV supports.
+We asume that readers have a basic understanding of Chainer framework (e.g. have a basic understanding of :class:`chainer.Link`).
+For users new to Chainer, please first read `Introduction to Chainer <https://docs.chainer.org/en/stable/tutorial/basic.html#write-a-model-as-a-chain>`_.
+
+In ChainerCV, we define the object detection task as a problem of, given an image, bounding box based localization and categorization of objects.
+ChainerCV supports the task by providing following features:
+
++ Visualization
++ BboxDataset
++ Detection Link
++ DetectionEvaluator
++ Training script for various detection models
+
+
+Bounding boxes in ChainerCV
+---------------------------
+Bounding boxes in an image are represented as a two-dimensional array of shape :math:`(R, 4)`,
+where :math:`R` is the number of bounding boxes and the second axis corresponds to the coordinates of bounding boxes.
+The coordinates are ordered in the array by :obj:`(y_min, x_min, y_max, x_max)`, where
+:obj:`(y_min, x_min)` and :obj:`(y_max, x_max)` are the :obj:`(y, x)` coordinates of the left bottom and the right bottom vertices.
+Notice that ChainerCV orders coordinates in :obj:`yx` order, which is the opposite of the convention used by other libraries such as OpenCV.
+This convention is adopted because it is more consistent with the memory order of an image that follows row-column order.
+
+Here is an example with a simple toy data.
+
+.. code-block:: python
+
+    import numpy as np
+    from chainercv.visualizations import vis_bbox
+    import matplotlib.pyplot as plt
+    img = np.zeros((3, 224, 224), dtype=np.float32)
```

Comment:
How about describing dtype? For example, "We use float32 as the dtype of bounding boxes. This type is suitable for many Chainer funtions/links."